setup.py

```diff
@@ -25,6 +25,7 @@
     'pyramid_duh>=0.1.1',
     'pyramid_jinja2',
     'pyramid_tm',
+    'rsa',
```

Comment:
I'm unfamiliar with this. Does the boto `create_signed_url` function depend on this module?